### PR TITLE
Add EndpointConfig.cs via nuget Content instead of install.ps1

### PR DESCRIPTION
**Pull request by bording**
connects to #68 

The powershell script in `install.ps1` that adds `EndpointConfig.cs` to the project was always adding the file, overwriting any existing file in the process.

This changes the nuget packages to use the Content section of nuget to add the file, which automatically correctly handles all of the selective adding/removing for us.

### CODE REVIEW

**Comment by SimonCropp** [comment]
this still breaks for 3 scenarios

 1. people who have renamed the file
 2. people who want to use the host extension points in helper library. so i will get a EndpointConfig in my lib when that lib is not a endpoint startable project
 3. people who move the file to another project to they can have a common shared startup code between multiple endpoints

**Comment by bording** [comment]
Sure, but those scenarios are all covered by one simple action: delete the EndpointConfig.cs file that gets added.

I have to make additional changes to projects after adding/updating nuget packages all the time, so I don't see this as being particularly onerous. If I don't want the defaults, then I know what I'm doing and can remove the extra files.

This PR doesn't make the situation any worse compared to the existing way we're doing it, and improves it in one significant way: We are no longer silently overwriting an existing file.

File: packaging/nuget/nservicebus.host.nuspec [hunk 1]
```diff
@@ -20,6 +20,7 @@
   </metadata>
   <files>
     <file src="..\..\binaries\NServiceBus.Host.???" target="lib\net452" />
-    <file src="..\..\packaging\nuget\nservicebus.host\*.*" target="tools" />
+    <file src="..\..\packaging\nuget\nservicebus.host\install.ps1" target="tools" />
+    <file src="..\..\packaging\nuget\nservicebus.host\EndpointConfig.cs.pp" target="Content" />
```
Comment (SimonCropp) [inline packaging/nuget/nservicebus.host.nuspec]: this should be 

    <file src="..\..\packaging\nuget\nservicebus.host\install.ps1" target="tools\net452" />
    <file src="..\..\packaging\nuget\nservicebus.host\EndpointConfig.cs.pp" target="Content\net452" />

**Comment by SimonCropp** [comment]
@bording fair enough. it is no more broken. smoke tested in an upgrade and an new project and it is ok

apart from the nuspec issue feel free to merge

File: packaging/nuget/nservicebus.host.nuspec (hunk 1 above)
Comment (bording) [inline packaging/nuget/nservicebus.host.nuspec]: Only lib had that before, but it makes sense to scope everything the same. Will push up that change.

Comment (SimonCropp) [inline packaging/nuget/nservicebus.host.nuspec]: yeah if we dont scope it i believe nuget will allow it to be installed on non 452 projects

**Comment by weralabaj** [comment]
@danielmarbach please, ping me directly when this is merged

**Comment by danielmarbach** [comment]
@weralabaj in

**Pull request merged**